android/settings.gradle

```diff
@@ -1,4 +1,6 @@
 rootProject.name = 'growupmobile'
+include ':react-native-image-picker'
+project(':react-native-image-picker').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-image-picker/android')
```

Comment:
Doesn't it autolink itself?

Reply:
No, it doesn't. I've tested it, everything works fine.